client/router_create.go

```diff
@@ -1161,6 +1161,9 @@ sasldb_path: /tmp/skrouterd.sasldb
 			if err != nil && !errors.IsAlreadyExists(err) {
 				return err
 			}
+			if err != nil {
+				fmt.Printf("aaa\n")
```

Comment:
Is this intended?

Reply:
🤦🏽 Do you need to ask? I thought I forcepushed it away, but apparently I did not.